packages/expo-calendar/android/src/main/java/expo/modules/calendar/CalendarModule.kt

```diff
@@ -181,7 +203,25 @@ class CalendarModule : Module() {
       }
     }
 
-    AsyncFunction("openEventInCalendar") { eventID: Int ->
+    AsyncFunction("createEventInCalendarAsync") Coroutine { eventOptions: CreatedEventOptions ->
+      val result = createEventLauncher.launch(eventOptions)
+      return@Coroutine result
+    }
+
+    AsyncFunction("openEventInCalendarAsync") Coroutine { params: ViewedEventOptions ->
+      val result = viewEventLauncher.launch(params)
+      return@Coroutine result
+    }
+
+    AsyncFunction("editEventInCalendarAsync") Coroutine { params: ViewedEventOptions ->
+      val result = viewEventLauncher.launch(params)
+      return@Coroutine result
+    }
+
+    /**
+     * @deprecated in favor of openEventInCalendarAsync
+     * */
```

Comment:
```suggestion
    /**
     * @deprecated Use [`openEventInCalendarAsync`](#openeventincalendarasyncparams-presentationoptions) instead.
     */
```

Deprecation note should be a valid sentence.

Reply:
this is kotlin, so it won't be in the docs

Reply:
Whoops! I went on comment spree it looks like! 😄